joomla/joomla-cms
# [4.0] replacing Bootstrap popover on com_content by already implemented css only version

Pull Request for Issue #33296 .

### Summary of Changes

This PR will replace the Bootstrap popover by the already implemented css only version of tooltip. 
```
<button blablabla aria-labelledby="unique-id">blabla</button>
<div id="unique-id" rel="tooltip">text</div>
```

### Testing Instructions

- Joomla 4 with blog sample data
- Go to Content > Articles and hover Stage, Featured or Status icon
- see the result
- Apply this PR and repeat step 2 and 3 

### Actual result BEFORE applying this Pull Request

![Schermafbeelding 2021-05-05 om 12 03 04](https://user-images.githubusercontent.com/639822/117125537-f19afd00-ad99-11eb-8e38-3db3c3f9334a.png)

When workflow is enabled the hover on Featured and Status is not working on my machine.

### Expected result AFTER applying this Pull Request

![Schermafbeelding 2021-05-05 om 11 35 47](https://user-images.githubusercontent.com/639822/117125557-f95aa180-ad99-11eb-9514-6c626b448896.png)
![Schermafbeelding 2021-05-05 om 11 35 51](https://user-images.githubusercontent.com/639822/117125559-f9f33800-ad99-11eb-8c7e-8bb79deb75b4.png)
![Schermafbeelding 2021-05-05 om 11 35 56](https://user-images.githubusercontent.com/639822/117125561-fa8bce80-ad99-11eb-9a51-9c677de6bcf1.png)


### Documentation Changes Required
